common/membership/grpc_resolver_test.go

```diff
@@ -100,7 +101,7 @@ func TestGRPCBuilder(t *testing.T) {
 		}),
 	)
 	require.NoError(t, err)
-
+	conn.Connect()
```

Comment:
I assume we do this in tests to force the connection to be made eagerly? And we don't have to do it in the regular code because grpc will connect on the first rpc?

Reply:
yes. That is my understanding.